### PR TITLE
Release Candidate for Version 2.1

### PR DESCRIPTION
**Pull request by ConstantinUp**
Current version of proposed changes for the release of 2.1.0. Reworked after VDA 5050 core team meeting.

Major changes are:
- Added **corridors**:
- Added **map distribution**:
- Added **Error hints** to AGV state:
- Added **Vehicle Configuration** to factsheet

### CODE REVIEW

File: assets/map_distribution_process.png
Comment (SusanneJunghans) [inline assets/map_distribution_process.png:1]: Should enableMap and deleteMap also report back the instant actions as finished?

File: assets/process_cancel_order.png
Comment (SusanneJunghans) [inline assets/process_cancel_order.png:1]: state "failed" should be all caps to be consistent with the remaining state names

File: assets/process_order_update.png
Comment (SusanneJunghans) [inline assets/process_order_update.png:1]: For the list of to-dos on the very left branch, list them with "-", too, to be consistent: 

- accept order
- set orderId and orderUpdateId
- populate states of new order (9)

File: json_schemas/order.schema
Comment (SusanneJunghans) [inline json_schemas/order.schema:1]: "... width in meters"

File: json_schemas/state.schema
Comment (SusanneJunghans) [inline json_schemas/state.schema:1]: Description of mapId: 
- apostrophe missing
"description": "ID of the map uniquely referencing a defined physical area of the vehicle's workspace." (suggestion to point out that it is clearly linked to a physically place).

Comment (SusanneJunghans) [inline json_schemas/state.schema:1]: Description of maps array: 
- no comma after map-objects

File: json_schemas/factsheet.schema
Comment (SusanneJunghans) [inline json_schemas/factsheet.schema:1]: Description of agvKinematic": simplified at start should be capital

Comment (SusanneJunghans) [reply json_schemas/factsheet.schema:1]: Same for "maxLoadMass", "localizationTypes", all entries in "properties"

File: json_schemas/state.schema
Comment (SusanneJunghans) [reply json_schemas/state.schema:1]: Description of mapStatus: should we talk about mapVersions here over maps?
"description": "Information on the status of the map indicating, if a map *version* is currently used on the vehicle. ENABLED: Indicates this map is currently active / used on the AGV. At most one map with the same mapId can have its status set to ENABLED.<br>DISABLED: Indicates this map *version* is currently not enabled on the AGV and thus could be enabled or deleted by request.",

File: assets/map_distribution_process.png
Comment (collani-bosch) [reply assets/map_distribution_process.png:1]: If it is mentioned in mapDownload, than also in enableMap and deleteMap.

File: assets/process_cancel_order.png
Comment (KIT-IFL) [reply assets/process_cancel_order.png:1]: FIxed in PR #292

File: assets/map_distribution_process.png
Comment (ConstantinUp) [reply assets/map_distribution_process.png:1]: This was fixed with new PR

File: assets/process_order_update.png
Comment (ConstantinUp) [reply assets/process_order_update.png:1]: Updated with new pull request

File: json_schemas/state.schema
Comment (ConstantinUp) [reply json_schemas/state.schema:1]: Changed to map version in this file and in markdown.

File: json_schemas/factsheet.schema
Comment (ConstantinUp) [reply json_schemas/factsheet.schema:1]: Will be fixed in Version 2.2

**Pull request closed without merging**